### ISSUE TITLE
Blocking to install other package maatwebsite/excel v3

### ISSUE THREAD

**Issue by nasirkhan** [yajra/laravel-datatables#1780]
### Summary of problem or feature request
I am using the `laravel-datatables` package and today I tried to install `maatwebsite/excel v3` but getting the error as following, is there any to resolve/bypass the situation? 

```
yajra/laravel-datatables-buttons v3.2.1 requires maatwebsite/excel ^2.0
```

### System details
- Operating System
- PHP Version
- Laravel Version
- Laravel-Datatables Version

**Comment by johnnemo** [comment]
Hello!
Maybe a temporary solution to this ? I face the very same issue.

Thank you

**Comment by yajra** [comment]
Yes, buttons 4.0 should fixed excel 3.0 compatibility and its compatible on v8. Should be stable as per my project, just updated to it recently with no errors or code adjustments from buttons v3.
